### PR TITLE
util/codec: add submodule for json

### PR DESCRIPTION
**Pull request by hhkbp2**
Hi,

This PR adds submodule for json document related codes. We could add json functions in dependent files into this submodule later.

PTAL @AndreMouche @andelf @lishihai9017

### CODE REVIEW

**Comment by AndreMouche** [comment]
LGTM

**Comment by AndreMouche** [comment]
@andelf @lishihai9017 PTAL

**Comment by lishihai9017** [comment]
LGTM

File: src/util/codec/mysql/json/mod.rs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2016 PingCAP, Inc.
```
Comment (siddontang) [inline src/util/codec/mysql/json/mod.rs]: 2017

**Pull request merged**